### PR TITLE
Adding colab example of input file generation

### PR DESCRIPTION
**Pull request by rosecers**
COSMO Retreat 2022, effort with @DavideTisi.

We have provided a minimal-example colab notebook for generating a chemiscope file. Goal is to be as user-friendly as possible and lower any barriers to that first sweet chemiscope.

### CODE REVIEW

**Comment by rosecers** [comment]
As in pin the versions? What would you suggest?

**Comment by rosecers** [comment]
@Luthaf test added

**Comment by rosecers** [comment]
@Luthaf I found the issue -- installing google colab causes a whole mess of problems. I added an in-elegant solution in the colab itself, let me know if you have any other suggestions that can go in the nbconvert call

**Comment by Luthaf** [comment]
Closing & reopening to trigger a new CI run

File: .github/workflows/colab.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    # Check all PR
+
+jobs:
+  pull-notebook:
+    runs-on: ubuntu-latest
+    steps:
+      - name: wget-notebook
+        uses: wei/wget@v1
+        with:
+          args: -O downloaded_file.ipynb 'https://docs.google.com/uc?export=download&id='1NU0gjtaHcB5Oc3NbFZiQYtctY2190hDu
```
[the remark this answers is not in the document]
Comment (rosecers) [reply .github/workflows/colab.yml:16]: Addressed in most recent edits!

File: .github/workflows/colab.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - name: wget-dataset
+        uses: wei/wget@v1
+        with:
+          args: -O frames.xyz 'https://rascaline--119.org.readthedocs.build/en/119/_downloads/c31f0a732fc71020e4e8e9b45d91c762/datasets.xyz'
```
[the remark this answers is not in the document]
Comment (rosecers) [reply .github/workflows/colab.yml]: Addressed in most recent edits!

**Pull request merged**